webapp/styles/AccountSettings.module.scss

```diff
@@ -0,0 +1,199 @@
+                }
+                .username {
+                    color: $red100;
+                }
+                .userEmail {
+                    color: $grey100;
+                }
+            }
+            .profileForm {
+                visibility: invisible;
+
+                .imgUploadContainer {
+                    width: 100%;
+                    display: flex;
+                    align-items: center;
+                    margin: 3em;
+                    border-bottom: 2px solid $grey400;
+                    margin-left: auto;
+                    padding-bottom: 35px;
+            
+                    .imgUpload {
+                        background-position: center;
+                        background-repeat: no-repeat;
+                        background-size: cover;
+                        height: 70px;
+                        width: 70px;
+                        display: flex;
+                        align-items: center;
+                        justify-content: center;
+                        border-radius: 50%;
+                        border: 2px solid #454545;
+                        transition: all 0.3s ease-in-out;
+                        cursor: pointer;
+                        margin-top: -37px;
+                        margin-bottom: 0;
+                        margin-left: 11px;
+
+                        &:hover {
+                            border: 2px solid #e5dada;
+                            box-shadow: 0 -1px 0 rgba(0, 0, 0, 0.04), 0 2px 4px rgba(0, 0, 0, 0.25),
+                            17px 3px 17px 17px $blue300;
+                        }
+
+                        input {
+                            display: none;
+                        }
+
+                        .imgUploadBox {
+                            display: flex;
+                            justify-content: space-between;
+                            align-items: center;
+                            position: relative;
+                            margin-right: 25px;
+                            width: 100%;
+                            height: 100%;
+                            border-radius: 50%;
+                            background-color: #454545;
+
+                            .userProfileImg {
+                                position: absolute;
+                                left: 0;
+                                top: 50%;
+                                min-width:  110px;
+                                min-height: 110px;
+                                max-width:  110px;
+                                max-height: 110px;
+                                transform: translateY(-50%);
+
+                                border: solid;
+                                border-width: 25px;
+                                background-image: 
+                                repeating-linear-gradient(0deg, $green100, $green100 100px, transparent 10px, transparent 20px), // left
```

Comment:
Investigate why the upload picture looks like this?
Prefer the upload picture not having random green circle and red triangles :((

![Screen Shot 2023-02-06 at 8 44 47 PM](https://user-images.githubusercontent.com/55214510/217150612-b7632a0c-290a-4587-b762-7d0af8a4ac85.png)

Reply:
actually it is exquisitely crafted.